app/models/ingredient.rb

```diff
@@ -1,3 +1,9 @@
 class Ingredient < ApplicationRecord
    validates_presence_of :name, :cost
+   has_many :recipe_ingredients
+   has_many :recipes, through: :recipe_ingredients
+
+   def self.sort_alphabetically
+      order(name: :asc)
```

Comment:
Nice encapsulation of this logic at the model level! 👍 Great MVC compliance

Reply:
When you write a model method, be sure to test it at the unit level in a model spec (i.e. ingredient_spec)